### PR TITLE
More changes to the introduction of step-6.

### PR DESCRIPTION
**Pull request by bangerth**
The next step for #6524.

### CODE REVIEW

File: examples/step-6/doc/intro.dox [hunk 1]
```diff
@@ -103,10 +103,42 @@ up with these &ldquo;hanging nodes&rdquo; if we do this.
...
+not refined, and consider for a moment the use of a bilinear finite
+element. In that case, the shape functions associated with the hanging
+nodes are defined in the obvious way on the two small cells adjacent
+to each of the hanging nodes. But how do we extend them to the big
+adjacent cells? Clearly, the function's extension to the big cell
+cannot be bilinear because then it needs to be linear along each edge
+of the large cell, and that means that it needs to be zero on the
+entire edge because it needs to be zero on the two vertices of the
+large cell on that edge. But it is not zero at the hanging node itself
+when seen from the small cells' side -- so it is not continuous.
```
Comment (davydden) [inline examples/step-6/doc/intro.dox]: in this case a picture would help, IMO. You can plot a single shape function (on vertex shared by coarse and refined cell) without distributing constraints.

**Comment by drwells** [comment]
@davydden Would you be okay with merging this without the picture? The patch, as-is, improves the introduction so I think we should leave that for later.

**Comment by davydden** [comment]
Sure

**Comment by bangerth** [comment]
Thanks. I will get back to @davydden 's request, though.

**Pull request merged**